### ISSUE TITLE
Create types for phone numbers

### ISSUE THREAD

**Issue by mconlon17** [openrif/vivo-isf-ontology#729]
In the previous version of vcard, there were not enough phone number types.  In particular, a voice line could not be asserted as such.  VIVO then used a "negative query" -- find all phone numbers without types -- and assumed they were voice lines.

The new vcard ontology may resolve this issue.  See #720

This is Jira https://jira.duraspace.org/browse/VIVO-1369

**Comment by mjaved495** [comment]
We need to upgrade to latest vcard ontology version. and thus I would not recommend to make this change. After we upgrade to latest version and still we see the need to subclasses we can do so.

**Comment by mconlon17** [comment]
Agree, Vcard upgrade needs to come first.  After the vcard upgrade is done (or more likely, during the vcard upgrade, this issue should be addressed.  The current function of VIVO for telephone numbers is a kludge that should be repaired -- requires the vcard upgrade, a simple code change to remove the current MINUS SPARQL query, very simple data change, and very simple changes to existing queries for phone numbers.
